src/helm/proxy/clients/openai_client.py

```diff
@@ -137,22 +140,21 @@ def do_it():
             embedding = response["data"][0]["embedding"]
         elif self._is_chat_model_engine(request.model_engine):
             for raw_completion in response["choices"]:
-                # The ChatGPT API doesn't support echo. If `echo_prompt` is true, combine the prompt and completion.
+                # The OpenAI API doesn't support echo. If `echo_prompt` is true, combine the prompt and completion.
```

Comment:
Thanks for fixing pre-existing typos.
nit: "OpenAI API" -> "OpenAI chat completion API" because the text completion APIs do give us do support these things.

Likewise for the other comments below.